app/templates/components/form-field.hbs

```diff
@@ -7,6 +7,7 @@
 {{else}}
   {{component type value=(get object propertyPath)
                    name=name
+                   options=options
```

Comment:
Rename to `componentProperties`, so it won't conflict with (for example) drop-down options